src/Components/Home/Home.css

```diff
@@ -0,0 +1,59 @@
+.header-des {
+    background-color: #0078d4;
+    padding: 12px 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: white;
+}
+
+.header-des a {
+    color: white;
+    font-size: 14px;
+    font-weight: 400;
+    text-decoration: none;
+}
+
+.header-des a:hover {
+    text-decoration: underline;
+}
+
+.home {
+    background-image: url('https://azurecomcdn.azureedge.net/cvt-59acfc73146e1c8512e35ae6c5e5518a5402dd994b33153cbd5853be39914df8/images/page/home/hero-desktop.png');
+    width: 100%;
+    height: 32rem;
+    background-size: cover;
+    display: flex;
+    justify-content: start;
+    align-items: center;
+}
+
+.home-section{
+    position: absolute;
+    z-index: 1;
+    margin: 0 15%;
+}
+
+.home .home-section h3 {
+    color: #50e6ff;
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 1.3;
+    margin: 0 0 9px;
+}
+
+.home .home-section h2 {
+    color: #f4f5f6;
+    font-size: 54px; 
+    line-height: 1.3;
+    font-weight: 500;
+}
```

Comment:
If we are changing the font-weight and size of **h1** and **h2** tags then there is no use of using them. Use a **div** or **span** and style it